Cog.py

```diff
@@ -0,0 +1,14 @@
+class Cog:
+    def __init__(self, bot, settings=None):
+        self.bot = bot
+        self.settings = settings
+
+        if self.settings is not None:
+            if self.__class__.__name__ in self.bot.config:
```

Comment:
Would have still liked to see self__class__.__name__ as a local variable to keep it clean.